lib/bgsv2/service.rb

```diff
@@ -132,6 +132,16 @@ def create_relationship(relationship_params)
       end
     end
 
+    def find_active_benefit_claim_type_increments
+      claims = []
+      with_multiple_attempts_enabled do
+        claims = service.benefit_claims.find_claims_details_by_participant_id(participant_id: @user.participant_id)
+      end
+      claims[:bnft_claim_detail].filter_map do |claim|
+        claim[:cp_claim_end_prdct_type_cd] if %w[CLR CAN].exclude?(claim[:status_type_cd])
+      end.uniq
```

Comment:
Will there ever be non-unique claims?

Reply:
Yes, the ep_code can be reused once the claim is closed. I *think* we won't get duplicates while we're filtering out the closed statuses (CLR CAN), but I'm not certain.